website/content/en/docs/best-practices/managing-resources.md

```diff
@@ -0,0 +1,187 @@
+---
+title: "Managing Resources"
+linkTitle: "Managing Resources"
+weight: 5
+description: This guide describes good practices concepts to Managing Resources.
+---
+
+## Managing Resources
+
+Note that pretty much all controllers consume:
+
+- `CPU`: largely based on the number of reconciliations they perform, which are generally related to event activity for resources they're watching.
+- `Memory`: largely based on the number of primary resources that exist (multiplied by some factor based on the number of operand resources they need to watch as a result) via informer caches.
+
+And then there is a concern that one Pod or Container could monopolize all available resources and Cluster admins must 
+consider the effects that one Pod or Container may have on other components. 
+
+In an effort to prevent a container from consuming all the resources on a cluster or affecting other workloads
+from being scheduled, many production clusters will define [ResourceQuota][k8s-resources-quotas] configurations.
+
+The [ResourceQuota][k8s-resources-quotas] configuration also applies to tenant workloads that are managed by
+your Operator. Cluster administrators will typically set a [ResourceQuota][k8s-resources-quotas] for each
+tenant's namespace as part of the onboarding. If a [LimitRange][k8s-limit-range] with default values
+has **not** been created in each namespace and your Operator creates Containers inside the tenant
+namespace without specifying at least resource requests for CPU and Memory of its Pods then,
+the system or quota may reject Pod creation with errors such as, for example: `[maximum cpu usage per Pod is 100m.  No limit is specified, maximum memory usage per Pod is 512Mi.  No limit is specified]`. 
```

Comment:
you're recommending people need to set `requests` but this error is complaining about missing `limits`

Reply:
Because the rule was for the max value. 
Removed the e.g. We do not need that.